### ISSUE TITLE
About Train model

### ISSUE THREAD

**Issue by defense81** [amazon-research/siam-mot#37]
I have trained the model many times , but  the gap about  3-4%  on MOT17 train set is always exist. I train  the model in the four gpus env,where  base lr 0.01  、steps=(20000,30000)  total iters are 35k, and max_size_train is 1200.
I want to know that  if the parameter of  max_size_train has a great impact on training ?  what's more, can you give me   some tips about training  the model ? 
   I appreciate your reply to my question. tahnks.

**Comment by bingshuai2019** [comment]
My initial guess is that the gap may comes from the fact you use "max_size_train == 1200".  I haven't tested it with model training on 4 GPUs, and I don't think that should be the major factor for the result discrepancy. It would be helpful if you train with the provided configurations configs/DLA_34_FPN_EMM_MOT17.yaml

**Comment by Leo63963** [comment]
I got the same question.
I trained with 2 GPUs with a learning rate of 0.0025,  max_iter is 50000, but got inferior performance in MOT17 training set. Just kindly ask, can I reproduce the results with my limited resources?
Plus, on configurations configs/DLA_34_FPN_EMM_MOT17.yaml, I found the TRAIN datasets have two files: crowdhuman_train_fbox and crowdhuman_val_fbox. Should we also add annotations of MOT17 for training?
Thanks

**Comment by Leo63963** [comment]
Hi @defense81 
Have you succeeded in reproducing the model?
Thanks.
